powerhome/playbook
# [PLAY-29] Remove chart background color

#### Screens

<img width="1524" alt="Screen Shot 2021-11-16 at 10 18 07 AM" src="https://user-images.githubusercontent.com/84349455/142021021-a0670004-abab-4aad-828a-f29de721a329.png">
<img width="1493" alt="Screen Shot 2021-11-16 at 10 18 22 AM" src="https://user-images.githubusercontent.com/84349455/142021033-3e1c188f-d2c4-4929-9635-1521d59570a7.png">
<img width="1518" alt="Screen Shot 2021-11-16 at 10 18 38 AM" src="https://user-images.githubusercontent.com/84349455/142021050-5f8d2b9a-2929-485f-8f80-90bebdf305b0.png">
<img width="1499" alt="Screen Shot 2021-11-16 at 10 18 51 AM" src="https://user-images.githubusercontent.com/84349455/142021069-3cbc9e6b-746d-4453-90da-99bf7b78f310.png">
<img width="1502" alt="Screen Shot 2021-11-16 at 10 19 11 AM" src="https://user-images.githubusercontent.com/84349455/142021081-cea5eea8-8456-4c2c-948e-133ad3cbb6b5.png">
<img width="1500" alt="Screen Shot 2021-11-16 at 10 19 23 AM" src="https://user-images.githubusercontent.com/84349455/142021091-ffed88bd-9882-4d94-8ee4-0f7773d44544.png">
<img width="1491" alt="Screen Shot 2021-11-16 at 10 20 00 AM" src="https://user-images.githubusercontent.com/84349455/142021120-e40fb91f-bd37-4f34-a78e-d207f68258e6.png">
<img width="1497" alt="Screen Shot 2021-11-16 at 10 19 46 AM" src="https://user-images.githubusercontent.com/84349455/142021107-09a3c864-da71-405e-a944-7dbc8bca9fda.png">

#### Breaking Changes

[Yes/No (Explain)]

No.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-29

#### How to test this

Pull up kits for Line Graph, Bar Graph, Circle Chart and Gauge, then make sure the background is not visible in both light and dark mode.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
